src/main/resources/swagger/api-docs.yaml

```diff
@@ -1974,10 +1974,17 @@ paths:
       summary: |
         deletes and rebuilds the entire index of published library datasets
       produces:
-      - text/plain
+        - text/plain
+        - application/json
       responses:
         200:
-          description: Success
+          description: List of errors if partial success, "()" if all datasets indexed successfully
```

Comment:
I could return a 200 if everything succeeded, and some other code, maybe a 202, if there were any failures - thoughts?

This isn't rawls - it's Elasticsearch. We use the bulk-index endpoint (https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-bulk.html), which reports an overall success with errors if any single documents within the batch failed to index.

Talked to you in person - we're gonna push this through as-is, since it's a developer convenience for now, and make it more robust later if we need to.